test/Driver/loaded_module_trace_multifile.swift

```diff
@@ -1,20 +0,0 @@
-// RUN: %empty-directory(%t)
-// RUN: %target-build-swift -emit-module -module-name Module %S/Inputs/loaded_module_trace_empty.swift -o %t/Module.swiftmodule
-// RUN: %target-build-swift -emit-module -module-name Module2 %S/Inputs/loaded_module_trace_empty.swift -o %t/Module2.swiftmodule
-// RUN: %target-build-swift %s %S/Inputs/loaded_module_trace_imports_module.swift -emit-loaded-module-trace-path %t/multifile.trace.json -emit-library -o %t/loaded_module_trace_multifile -I %t
-// RUN: %FileCheck %s < %t/multifile.trace.json
-
-// This file only imports Module2, but the other file imports Module: hopefully they both appear!
-
-// CHECK: {
-// CHECK: "name":"loaded_module_trace_multifile"
-// CHECK: "arch":"{{[^"]*}}"
-// CHECK: "swiftmodules":[
-// CHECK-DAG: "{{[^"]*\\[/\\]}}Module2.swiftmodule"
-// CHECK-DAG: "{{[^"]*\\[/\\]}}Module.swiftmodule"
-// CHECK-DAG: "{{[^"]*\\[/\\]}}Swift.swiftmodule{{(\\[/\\][^"]+[.]swiftmodule)?}}"
-// CHECK-DAG: "{{[^"]*\\[/\\]}}SwiftOnoneSupport.swiftmodule{{(\\[/\\][^"]+[.]swiftmodule)?}}"
-// CHECK: ]
-// CHECK: }
-
-import Module2
```

Comment:
What happened to this test?

Reply:
From my PR message:

> I have deleted the local_module_trace_multifile test because it is not testing anything additional over loaded_module_trace -- the latter already checks for direct imports in swiftmodules-detailed-info.

The test was trying to check that transitive dependencies show up under `swiftmodules`, but we're already testing that in `loaded_module_trace.swift`.

Reply:
It is testing something additional: dependencies from multiple files in the same module.